intro-to-git/pull-requests.qmd

```diff
@@ -0,0 +1,135 @@
+---
+title: "Pull requests"
+---
+
+> 🎯 Goal: By the end of this chapter, you'll have understand the basic process of reviewing and merging pull requests in Github.
+
+## Suggest and Review Changes with Pull Requests
+
+**Pull Requests** (often abbreviated to just "PR" in the programming world) are GitHub’s way of saying, “Hey, I’ve made some changes. Can we look them over and maybe merge them into the main project?”
+
+Even if you’re working solo, they’re a great habit because they give you a space to:
+
+-   Review your changes before merging.
+
+-   Track a discussion about the changes you're making with your collaborators.
+
+-   Ensure everything looks good before merging.
+
+We'll walk through the basic pull request workflow below.
+
+## Step 1: Create a Branch and push to Github
+
+First, make sure you’re on a new branch (just like we did in [Working with Feature Branches](/intro-to-git/working-with-branches.html))
+
+1.  Create a new feature branch by running: `bash     git checkout -b improve-plot`
+
+2.  Make some edits to your plot file(e.g., tweak a plot or add new text)
+
+3.  Stage and commit the changes:
+
+    ``` bash
+    git add filename.qmd 
+    git commit -m "Improve plot aesthetics"
+    ```
+
+4.  Push the branch to GitHub:
+
+``` bash
+git push -u origin improve-plot
+```
+
+## Step 2: Open pull request (PR) on GitHub
+
+Instead of merging our changes from the command like like we did in the last chapter, we're going to handle the merging process in Github so we can take advantage of some of the nice features it offers for code reviews.
+
+Once your branch is on GitHub, you’ll see a banner like
+
+![Github repo Pull Request banner](../images/pr-banner-example.png)
+
+To begin the pull request process:
+
+1.  Click the blue “Compare & pull request” button! (Tip: You can also find this button under the “Pull requests” tab at the top.)
```

Comment:
In the Tip, I would also note the banner isn't always there if there is no recent commit. Maybe set this off visually somehow, too?